charts/federation-v2/README.md

```diff
@@ -0,0 +1,101 @@
+# Kubernetes Federation V2
+
+Kubernetes Federation V2 is a Kubernetes Incubator project. It builds on the sync controller
+(a.k.a. push reconciler) from [Federation v1](https://github.com/kubernetes/federation/)
+to iterate on the API concepts laid down in the [brainstorming
+doc](https://docs.google.com/document/d/159cQGlfgXo6O4WxXyWzjZiPoIuiHVl933B43xhmqPEE/edit#)
+and further refined in the [architecture
+doc](https://docs.google.com/document/d/1ihWETo-zE8U_QNuzw5ECxOWX0Df_2BVfO3lC4OesKRQ/edit#). 
+Access to both documents is available to members of the
+[kubernetes-sig-multicluster google
+group](https://groups.google.com/forum/#!forum/kubernetes-sig-multicluster).
+
+## Prerequisites
+
+- Kubernetes 1.11+
+- Helm 2.10+
+
+## Installing the Chart
+
+First you'll need to create the reserved namespace for registering clusters with the
+cluster registry:
+
+```bash
+kubectl create ns kube-multicluster-public
+```
+
+Install the chart with the release name `federation-v2`:
+
+```bash
+$ helm install . --name federation-v2 --namespace federation-system
```

Comment:
this command fails when executed from root of the repo as follows:
```
root@shashi-laptop:~/gopath/src/github.com/kubernetes-sigs/federation-v2# helm install . --name federation-v2 --namespace federation-system
Error: no Chart.yaml exists in directory "/root/gopath/src/github.com/kubernetes-sigs/federation-v2
```
So the command should be
```
helm install charts/federation-v2 --name federation-v2 --namespace federation-system
```

Reply:
Done. Actually, if switch to `charts/federation-v2`, then we will be able to install it with `.`, but using whole dir is more easy to understand.